doc/mgr/administrator.rst

```diff
@@ -44,12 +44,12 @@ Calling module commands
 Where a module implements command line hooks, using the Ceph CLI's
 ``tell`` command to call them like this::
 
-    ceph tell mgr <command | help>
+    ceph tell mgr <command | get_command_descriptions>
```

Comment:
get_command_descriptions is indeed not meant for humans.  It's what the CLI uses to resolve a series of positional arguments into a {"prefix":...} dict for use in commands.